inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.html

```diff
@@ -56,6 +56,10 @@ <h4 class="list-group-item-heading">
             <div class="text-right">
               <i wicket:enclosure="created">Created: <wicket:container wicket:id="created"/></i>
             </div>
+            <div>
+          	  <input wicket:id="leaveProject" type="button" class="btn btn-primary" wicket:message="value:leaveDialog.title"/> 
+        	</div>
+        	<div wicket:id="confirmLeave"></div>
```

Comment:
Does the confirmation dialog have to reside in the repeater? Wouldn't it be better to have it once outside of the list?